content/agent/faq/send-logs-and-configs-to-datadog-via-flare-command.md

```diff
@@ -1,58 +1,65 @@
 ---
-title: Send logs and configs to Datadog via flare command
+title: Send logs and configs to Datadog via flare
 kind: faq
 further_reading:
 - link: "/agent/"
   tag: "Documentation"
   text: Learn more about the Datadog Agent
 ---
 
-## Linux and Mac OS X
+## Overview
 
-If you are running the 5.3 version of the agent, you're able to send all necessary troubleshooting information to our Support Team, with one command!
+If you are running the 5.3 version of the agent (or higher), you're able to send all necessary troubleshooting information to our Support Team, with one flare command!
```

Comment:
running version 5.3 (or higher) of the agent,

Reply:
👍